### ISSUE TITLE
config file generates error

### ISSUE THREAD

**Issue by johndeighan** [DanielXMoore/Civet#1480]
Just getting started with civet, so I started creating a civet.config.js file (I'm done trying to edit JSON files):

```
export default {
    parseOptions: {
        
    }
}

```
As you can see, I haven't added any options yet. My main.civet file is equally simple (and probably not relevant):

`console.log "OK"
`
However, when I try to esecute main.civet, I get a nasty error message:

`Error: Civet config file must export an object, not null
`
I'm clearly exporting an object, so you should either allow an empty set or options or improve the error message. FYI, when I added an option, I no longer got the error message. Here is the full message in case that's useful:

```
$ civet main.civet
C:\Users\johnd\AppData\Roaming\npm\node_modules\@danielx\civet\dist\config.js:7331
    throw new Error(`Civet config file must export an object, not ${Array.isArray(data) ? "array" : data != null ? typeof data : "null"}`);
          ^

Error: Civet config file must export an object, not null
    at loadConfig (C:\Users\johnd\AppData\Roaming\npm\node_modules\@danielx\civet\dist\config.js:7331:11)
    at async cli (C:\Users\johnd\AppData\Roaming\npm\node_modules\@danielx\civet\dist\civet:515:20)

Node.js v22.5.1

```

**Comment by edemaine** [comment]
That definitely sounds like a bug. Thanks for reporting! Will take a look tomorrow.

**Comment by edemaine** [comment]
I can't reproduce your issue. Which version of Civet are you using? (`civet --version`) Could you try updating?

![image](https://github.com/user-attachments/assets/ee81c00b-8063-45cc-ac00-684178924a4d)

**Comment by johndeighan** [comment]
Using version 0.8.4. I'll try upgrading and test again. Thanks for looking into it.

**Comment by johndeighan** [comment]
Congratulations, it looks like this issue is fixed in version 0.8.7
